src/activate/activate.c

```diff
@@ -332,33 +338,33 @@ static int help(void) {
                "     --fdname=NAME[:NAME...] Specify names for file descriptors\n"
                "     --inetd                 Enable inetd file descriptor passing protocol\n"
                "\nNote: file descriptors from sd_listen_fds() will be passed through.\n"
-               "\nSee the %s for details.\n"
-               , program_invocation_short_name
-               , link
-        );
+               "\nSee the %s for details.\n",
+               program_invocation_short_name,
+               link);
```

Comment:
hmm, so we place things like this in this case since we want to minimize diffs, when the help text is updated, i.e. in order to minimize context in each line. It would be sad to give that up

Reply:
hmm, i dont see a way to programatically distinguish this case from "normal functions", which should be freely wrapped and have the comma right aligned to the arguments.

Anyway this might be a rare case where the clang-format should better just simply be turned off.